influence_benchmark/RL/base_iteration.py

```diff
@@ -26,23 +26,29 @@ def __init__(
         training_args: dict,
         accelerate_config_path: str,
         script_path: str,
-        model_name: str,
+        agent_model_name: str,
+        env_model_name: str,
         n_trajs_per_initial_state: int,
         iterations: int,
         top_n_trajs_per_initial_state: int = 1,
         run_name: Optional[str] = None,
         devices: Optional[list] = None,
         mode: str = "multi",
         log_to_wandb: bool = False,
+        final_reward: bool = False,
+        override_initial_traj_path=None,
```

Comment:
I don't think this is used anywhere, and it kind of complicates the logic in various places. Do we want to keep it?

Reply:
I use it when I accidentally mess up the training code such that it crashes after generating the first iteration of trajectories